.github/workflows/release.yml

```
@@ -255,7 +255,7 @@ jobs:
         with:
           context: .
           file: .github/workflows/deps/prebuilt.Dockerfile
-          push: true
+          push: ${{ github.event.pull_request.head.repo.full_name == github.repository && 'true' || 'false' }}
```

it should be enough to do this, I think?

```suggestion
          push: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```